src/scripts/main.js

```diff
@@ -1,3 +1,84 @@
 'use strict';
 
-// write code here
+const buttons = document.querySelector('.container');
+const table = document.querySelector('.field');
+
+buttons.addEventListener('click', (e) => {
+  const button = e.target.closest('.button');
```

Comment:
Consider checking if `button` is not null before accessing its properties. This will prevent runtime errors if the click event is triggered on an element that is not a button.